README.md

```diff
@@ -36,7 +36,7 @@ The matchmaker template receives intents with the borsh encoding define in `data
 
 ```shell
 # Build the provided validity predicate, transaction and matchmaker wasm modules
-docker build -t anoma-wasm wasm
+make build-wasm-image-docker
 make build-wasm-scripts-docker
```

Comment:
I guess this can then be just
```suggestion
make build-wasm-scripts-docker
```

Reply:
hello tomas,
 
 yes, sure - i just keept the image-build step in similarity to the content before the change :)
 
 br,
 joachim